Makefile

```diff
@@ -67,3 +67,8 @@ ruff:
 	ruff ${SOURCE_DIRS}
 
 check: ruff yapf mypy test
+
+build-docs:
+	sphinx-apidoc -f -M -e -T -d 3 mantidimaging **/test **/test_helpers **/eyes_tests -o docs/api/
+	sphinx-build ./docs ./docs/build
+	sphinx-multiversion ./docs docs/build/
```

Comment:
I think these need to be
```
sphinx-build ./docs ./docs/build/html/
sphinx-multiversion ./docs docs/build/html/
```
To match the old layout, otherwise I think the publish step may fail.